### PR TITLE
Don't create additional jar files in the root folder #377

### LINKED ISSUE #377: Don't create additional jar files in the root folder
We should try to keep the root folder as clean as possible

### PR DESCRIPTION
**Pull request by HansNewbie**
Fixes #377

### CODE REVIEW

File: src/main/resources/updater/Instruction to use past versions.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Copy the relevant version JAR to the root folder (no need to rename it) and run that JAR file.
```
Comment (HansNewbie) [inline src/main/resources/updater/Instruction to use past versions.txt]: Is this instruction to use backup versions enough? What happens is all backup version will be in `past_versions/`. This `instruction.txt` will be in that folder as well.

To use a backup version, user just need to copy that JAR backup to the root and run that file.

Comment (damithc) [inline src/main/resources/updater/Instruction to use past versions.txt]: `If the latest version does not work for some reason, you may want to use a past version until the latest version is fixed. In such a case, you can copy the JAR file of the version you want to use to the root folder (no need to rename it) and run that JAR file.`

Comment (HansNewbie) [inline src/main/resources/updater/Instruction to use past versions.txt]: Should this instruction be put in the root folder or inside `past_versions/`? I am concerned that putting it in `past_versions/` might hide it from users, but putting it in the root makes the root folder dirty.

Comment (damithc) [inline src/main/resources/updater/Instruction to use past versions.txt]: Yes, we can put it in the root folder too.

Comment (HansNewbie) [inline src/main/resources/updater/Instruction to use past versions.txt]: Noted.

**Comment by HansNewbie** [comment]
Will merge if no more comments.

**Pull request merged**